src/components/CancelButton/CancelButton.css

```diff
@@ -0,0 +1,18 @@
+.CancelButton {
+  width: 56px;
```

Comment:
we still need to work on the units... from px to em. my guess is we shall do this in the refactor